### ISSUE TITLE
Example for generating wallet address from xpub

### ISSUE THREAD

**Issue by damascene** [meherett/python-hdwallet#4]
I would like to request an example showing how we can generate addresses from extended public key (xpub).

**Comment by meherett** [comment]
@damascene Currently, You can't generate addresses but this month I will release the 1.0.0 version, your question is also one future on this v1.0.0. So, Just be patient for a few 2-3 weeks, bro.

**Comment by hub-il** [comment]
I'm also interested in seeing an example.  It would be great if it was possible to do something similar to below where I could recreate the addresses in a HD wallet so that I could verify that an address was derived from the xpub.  

~~~~
./bitcointool -command hdderive -keypath m/1 -privkey xprv9s21ZrQH143K3C5hLMq2Upsh8mf9Z1p5C4QuXJkiodSSihp324YnWpFfRjvP7gqocJKz4oakVwZn5cUgRYTHtNRvGqU5DU2Gn8MPM9jHvfC
~~~~

**Comment by meherett** [comment]
@hub-il  That's why I didn't close this issue. I'm also working on it, bro :)

**Comment by meherett** [comment]
Hello @damascene & @hub-il, Now, It's implemented on this v1.2.0 package. Thanks :)
